src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
![image](https://user-images.githubusercontent.com/96409563/212544390-7345b61e-1db1-400d-a915-48740d9d2506.png)
![image](https://user-images.githubusercontent.com/96409563/212544399-4c1af6e1-7d01-421d-a5ff-4505c305f5c8.png)